### ISSUE TITLE
Incompatible with EdB Prepare Carefully 1.8.2-1

### ISSUE THREAD

**Issue by Famous-Shoes** [Famous-Shoes/Battle-Rattle#47]
The following error is thrown when Battle Rattle is loaded and one tries to switch to the EdB Prepare Carefully mode for character creation:

```
Swapped in EdB Prepare Carefully Character Creation Page
(Filename: C:/buildslave/unity/build/artifacts/StandalonePlayerGenerated/UnityEngineDebug.cpp Line: 56)

Cannot find matching layer for apparel.  Last layer: Accessory
(Filename: C:/buildslave/unity/build/artifacts/StandalonePlayerGenerated/UnityEngineDebug.cpp Line: 56)

Exception filling window for EdB.PrepareCarefully.Page_CharMaker: System.ArgumentOutOfRangeException: Argument is out of range.
Parameter name: index
  at System.Collections.Generic.List`1[Verse.Graphic].CheckIndex (Int32 index) [0x00000] in <filename unknown>:0 
  at System.Collections.Generic.List`1[Verse.Graphic].set_Item (Int32 index, Verse.Graphic value) [0x00000] in <filename unknown>:0 
  at EdB.PrepareCarefully.PawnFacade.InitializeWithPawn (Verse.Pawn pawn) [0x00000] in <filename unknown>:0 
  at EdB.PrepareCarefully.PawnFacade..ctor (Verse.Pawn pawn) [0x00000] in <filename unknown>:0 
  at EdB.PrepareCarefully.PrepareCarefully.Initialize () [0x00000] in <filename unknown>:0 
  at EdB.PrepareCarefully.Page_CharMaker.FillWindow (Rect inRect) [0x00000] in <filename unknown>:0 
  at Verse.Layer_Window.DoWindowContents () [0x00000] in <filename unknown>:0 
```

**Comment by Famous-Shoes** [comment]
This appears to be because of Battle Rattle's use of the new Accessory layer provided by Alpha 10; EdB Prepare Carefully's code doesn't seem to handle that correctly. 

_But_, the mod Stuff for Haul also uses this new Accessory layer and does not (yet) seem to trigger this symptom in EdB Prepare Carefully, so there must be a work-around that I'm just not seeing.

**Comment by Famous-Shoes** [comment]
Resolved by fixes in EdB Prepare Carefully 1.8.3.
